### DISCUSSION TITLE
Смена подсети Wireguard

### DISCUSSION THREAD

**Discussion by crubik** [xtrime-ru/antizapret-vpn-docker#68] [Q&A] [answered]
Коллеги, проверьте пожалуйста еще раз обновления WG
Дальше WG трафик не уходит. после смены пула адресов с 10.8xxx на 10.1.xx.xx 
![Screenshot_3](https://github.com/user-attachments/assets/615c3021-cdaf-414b-97f6-a0e4afef579f)

хотя до WG коннект есть

**Comment by Dimariqe** [comment]
Уточните до какого IP вы хотите достучаться? Какие `WG_DEFAULT_ADDRESS` и `WG_ALLOWED_IPS` у вас установлены? Если вы меняли `WG_DEFAULT_ADDRESS`, то необходимо в `WG_ALLOWED_IPS` изменить подсеть `10.1.166.0/24,` на ту, которую вы поменяли.
Так же есть вероятность, что вы используете старые conf файлы, обновите на новые.
PS: Если проблема после этого решилась, переименуйте пожалуйста дискуссию, что бы другим было проще разобраться с проблемой, к примеру на "Смена подсети Wireguard"

**Comment by crubik** [comment]
на данный момент работает версия v3.3.0
поднят на vps Антизапрет и WG. Всё работает отлично
на клиенте стоит mikrotik, который резолвит через WG определенный сайты, в т.ч. ютуб
![Screenshot_1](https://github.com/user-attachments/assets/321bb7a4-5f21-4e43-b164-b595762ffcf7)
![Screenshot_2](https://github.com/user-attachments/assets/bd8fdb0a-1302-4a7e-a59a-9e820625367d)

ютуб отлично работает и трассировка проходит через vps
![Screenshot_3](https://github.com/user-attachments/assets/55863ed7-06b9-4108-b42f-04e06aaaafff)

т.к. я после обновления контейнеров в докере ловлю ошибку WARN[0000] , то выполняю принудительный тормоз контейнера wg
![Screenshot_5](https://github.com/user-attachments/assets/aca66ce3-3a67-4d1b-a945-b9c2c4a5ff40)
![Screenshot_6](https://github.com/user-attachments/assets/979a6c6e-4a63-41b4-9227-6fd7a1dcf3f1)

Вроде всё ок. Запустилось. Веб заработал и остались старые клиенты. НО. не пускает на ютуб
![Screenshot_7](https://github.com/user-attachments/assets/c1549e1e-f2c6-4ace-b22c-d061584f2f68)
![Screenshot_8](https://github.com/user-attachments/assets/a19dfae8-7246-4fd9-9360-274aa940170e)

Ок. Удаляю клиента и создаю новый
Новый клиент с новой подсетью. Ну ок
![Screenshot_9](https://github.com/user-attachments/assets/0e1032f4-51a1-4fd8-a950-f949aa874adc)

Коннект с WG на vps есть, но дальше - нет
![Screenshot_10](https://github.com/user-attachments/assets/4b19a964-df5d-40bd-bbb0-62795c05466c)


Выяснил. Amnesia не работает с микротиками. там протокол WG фикшен и штатный клиент микротика его не принимает. Обычный WG работает
Ошибка была в подсеnях. Висела старая сеть на интерфейсе WG. Сменил - все заработало
![Screenshot_12](https://github.com/user-attachments/assets/a059e658-ce39-4132-a0ef-68082af32874)

**Comment by Dimariqe** [comment, marked as answer]
10.8.0.1 убран в WG, DNS теперь в подсети antizapret. Ранее вообще очень много локальных адресов занимал WG. Но у меня к примеру много локальных сетей, а 10.8.0.0/24 вообще занята модемом 4G. Что бы минимизировать эти конфликты, я уменьшил диапазоны до минимальных, задал не популярную подсеть для WG:
10.224.0.0/15 - Подсеть антизапрета, DNS теперь 10.224.0.1
10.1.166.0/24 - Подсеть WG
Но с флагом `FORCE_FORWARD_DNS=true` любой IP c маршрутом через WG будет отвечать DNSом антизапрета.

Судя по скриншотам веб панельки у вас на сервере не Amnezia, а обычный WG.

**Reply by crubik** [reply]
Амнезия было протестирована и удалена, т.к. смысла ее показывать нет) Если бы работало - всё через нее сделал бы

**Reply by Dimariqe** [reply]
Ну тогда все верно :)
Итого для решения проблемы:
обновите конфиги из веб панельки
если были маршруты заданы статично - проверьте их, на данный момент необходимы только 2:
`10.224.0.0/15`
`10.1.166.0/24`
Если установлен DNS в ручную - проверьте что он 10.224.0.1

**Reply by crubik** [reply]
lда, всё хорошо. ДНС использую DoH 
С маршрутами всё ок, проверил
